src/main/kotlin/org/rust/RsProjectTaskQueueService.kt

```diff
@@ -249,4 +245,8 @@ private class RsBackgroundTaskQueue {
             data class Running(val indicator: ProgressIndicator) : State()
         }
     }
+
+    companion object {
+        private val LOG = Logger.getInstance(RsBackgroundTaskQueue::class.java)
```

Comment:
nit: Note, there is `logger` generic function to use reified generic param instead of `RsBackgroundTaskQueue::class.java`